website/content/en/docs/setup/installation/platforms/kubernetes.md

```diff
@@ -50,12 +50,12 @@ namespace: vector
 
 bases:
   # Include Vector recommended base (from git).
-  - github.com/timberio/vector/distribution/kubernetes/vector-agent?ref=v{{< version >}}
+  - github.com/vectordotdev/vector/distribution/kubernetes/vector-agent?ref=v{{< version >}}
 
 images:
   # Override the Vector image to avoid use of the sliding tag.
-  - name: timberio/vector
-    newName: timberio/vector
+  - name: vectordotdev/vector
+    newName: vectordotdev/vector
```

Comment:
```suggestion
  - name: timberio/vector
    newName: timberio/vector
```
These refer to the docker repo

Reply:
👍 just for reference, renaming the docker org is mentioned in https://github.com/vectordotdev/vector/issues/9137